docs/sources/tutorials/quick-start-browser.md

```diff
@@ -5,71 +5,93 @@ This document describes how to set up and use Grafana Faro Web SDK. For more inf
-traces:
-  configs:
-    - name: default
-      remote_write:
-        - endpoint: tempo-us-central1.grafana.net:443
-          basic_auth:
-            username: xxx
-            password: xxx
-      receivers:
-        otlp:
-          protocols:
-            grpc:
-            http:
-              cors:
-                allowed_origins:
-                  - http://localhost:1234
-                max_age: 7200
-integrations:
-  app_agent_receiver_configs:
-    - autoscrape:
-        enable: true
-        metrics_instance: 'default'
-      instance: 'frontend'
-      logs_instance: 'default'
-      traces_instance: 'default'
-      server:
-        host: 0.0.0.0
-        port: 12345
-        api_key: 'secret' # optional, if set, client will be required to provide it via x-api-key header
-        cors_allowed_origins:
-          - 'https://my-app.example.com'
-      logs_labels: # labels to add to loki log record
-        app: frontend # static value
-        kind: # value will be taken from log items. exception, log, measurement, etc
-      logs_send_timeout: 5000
-      sourcemaps:
-        download: true # will download source file, extract source map location,
-        # download source map and use it to transform stack trace locations
+agent receiver integration, refer to [app_agent_receiver_config][grafana-alloy-receiver-config].
+
+```river
+prometheus.remote_write "metrics_write" {
+    endpoint {
+        name = "default"
+        url  = <remote_write_url>
+        queue_config { }
+        metadata_config { }
+    }
+}
+
+loki.process "logs_process_client" {
+    forward_to = [loki.write.logs_write_client.receiver]
+
+    stage.logfmt {
+        mapping = { "kind" = "", "service_name" = "", "app" = "" }
+    }
+
+    stage.labels {
+        values = { "kind" = "kind", "service_name" = "service_name", "app" = "app" }
+    }
+}
+
+loki.write "logs_write_client" {
+    endpoint {
+        url = <loki_write_url>
+    }
+}
+
+logging {
+    level = "info"
+}
+
+faro.receiver "integrations_app_agent_receiver" {
+    server {
+        listen_address           = "0.0.0.0"
+        listen_port              = 12345
+        cors_allowed_origins     = ["https://my-app.example.com"]
+        api_key                  = my_super_app_key
+        max_allowed_payload_size = "10MiB"
+
+        rate_limiting {
+            rate = 100
+        }
+    }
+
+    sourcemaps { }
+
+    output {
+        logs   = [loki.process.logs_process_client.receiver]
+        traces = [otelcol.exporter.otlp.trace_write.input]
+    }
+}
+
+otelcol.receiver.otlp "default" {
+    grpc {
+        include_metadata = true
+    }
+
+    output {
+        metrics = []
+        logs    = []
+        traces  = [otelcol.exporter.otlp.trace_write.input]
+    }
+}
+
+otelcol.exporter.otlp "trace_write" {
+    retry_on_failure {
+        max_elapsed_time = "1m0s"
+    }
+
+    client {
+        endpoint = <tempo_endpoint>
+
+        tls {
+            insecure = true
```

Comment:
@bubu11e I have a last one.

The [tls.insecure setting is optional](https://grafana.com/docs/alloy/latest/reference/components/otelcol.exporter.otlp/#tls-block) and we shouldn't set it to insecure by default.
User should explicitly decide for this setting if they need to.

Reply:
Would be better indeed. Thanks and Fixed.